spec

```diff
@@ -4443,7 +4443,7 @@ beginseg
     name "ovl_Obj_Jg_Gakki"
     compress
     include "build/src/overlays/actors/ovl_Obj_Jg_Gakki/z_obj_jg_gakki.o"
-    include "build/data/ovl_Obj_Jg_Gakki/ovl_Obj_Jg_Gakki.data.o"
+    /* include "build/data/ovl_Obj_Jg_Gakki/ovl_Obj_Jg_Gakki.data.o" */
     include "build/data/ovl_Obj_Jg_Gakki/ovl_Obj_Jg_Gakki.reloc.o"
```

Comment:
Since you already imported data, you can remove the `.data.o` line.
Also, since you have the actor completely matching, you need to change the reloc line to use the one we are generating instead of the extracted one
```suggestion
    include "build/data/ovl_Obj_Jg_Gakki/ovl_Obj_Jg_Gakki_reloc.o"
```

Reply:
No, should be `build/overlays/actors/ovl_Obj_Jg_Gakki/ovl_Obj_Jg_Gakki_reloc.o`, I think: it's in the same build folder as the `.o` file, but named after the overlay.

Reply:
Sorry, I got the path wrong, it should be like this:
```suggestion
    include "build/src/overlays/actors/ovl_Obj_Jg_Gakki/ovl_Obj_Jg_Gakki_reloc.o"
```

Reply:
Okay, I get it now. Thank you